stats_overlay.js

```diff
@@ -0,0 +1,259 @@
+
+
+
+/** @constructor */
+shaka.StatsOverlay = function() {
+
+  /** @private {?shaka.player.Player} */
+  this.player_;
+
+  /** @private {?HTMLElement} */
+  this.canvas_;
+
+  /** @protected {number} */
+  this.width = 200;
+
+  /** @protected {number} */
+  this.height = 100;
+
+  /** @private {?number} */
+  this.startTimestamp;
```

Comment:
The linter had plenty to say but not about that. I have updated the private variables and functions to be more inline with the rest of the project.